lib/peep.rb

```diff
@@ -0,0 +1,26 @@
+
+require 'pg'
+
+class Peep
+
+  def self.all
+    if ENV['ENVIRONMENT'] = 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    result = connection.exec("SELECT * FROM peeps;")
+    result.map { |peep| peep['content'] }
+  end
+
+  def self.create(content:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
+    connection.exec("INSERT INTO peeps (content) VALUES('#{content}')")
```

Comment:
this line fails if the peep `content` contains an apostrophe, beacause it interprets the apostrophe as ending the VALUES string. see if you can work out a way to fix this :)